File: AppBank/app/src/main/java/com/dev/appbank/model/StatementList.java
package com.dev.appbank.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class StatementList implements Serializable {

    private static final long serialVersionUID = 1L;
    @SerializedName("statementList")
    private List<Statement> statementList;

    public StatementList() {

    }

    public StatementList(List<Statement> statementList) {
        this.statementList = statementList;
    }

    public List<Statement> getStatementList() {
        return statementList;
    }

    public void setStatementList(List<Statement> statementList) {
        this.statementList = statementList;
    }

}

File: app/src/main/java/com/example/bankapp/pojo/statement/StatementList.kt
package com.example.bankapp.pojo.statement

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class StatementList {

    @SerializedName("title")
    @Expose
    var title: String? = null
    @SerializedName("desc")
    @Expose
    var desc: String? = null
    @SerializedName("date")
    @Expose
    var date: String? = null
    @SerializedName("value")
    @Expose
    var value: Double? = null

}
